src/QMCWaveFunctions/BsplineFactory/SplineC2C.cpp

```diff
@@ -167,7 +167,8 @@ inline void SplineC2C<ST>::assign_v(const PointType& r,
                                     int last) const
 {
   // protect last
-  last = last > kPoints.size() ? kPoints.size() : last;
+  const size_t last_real = std::min(kPoints.size(), psi.size());
+  last = last > last_real ? last_real : last;
```

Comment:
Should be called last_cplx as the last complex output orbital.

Reply:
done